valohai_cli/commands/yaml/step.py

```diff
@@ -25,34 +26,35 @@ def step(filenames: List[str]) -> None:
     :param filenames: Path(s) of the Python source code files.
     """
     project = get_project()
+    project_yaml = project.get_yaml_path()
```

Comment:
Does this need to be a separate variable?

Doesn't just

```suggestion
    yaml = yaml or project.get_yaml_path()
```
do to override the falsy default with whatever's configured in the project?